model-optimizer/CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 # Copyright (C) 2018-2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+if (NOT NGRAPH_PYTHON_BUILD_ENABLE)
+    message(WARNING "Please enable nGraph Python API (_pyngraph) target to enable Model Optimizer target")
+elseif(NOT ENABLE_PYTHON)
```

Comment:
can we enable NGRAPH_PYTHON_BUILD_ENABLE by default if ENABLE_PYTHON is ON?

Reply:
NGRAPH_PYTHON_BUILD_ENABLE requires some more variables than ie_api.